src/routes/tools/wildfire/_helpers.js

```diff
@@ -22,3 +23,9 @@ export const getMapImages = ({
       palette,
     })
   );
+
+export const getAvgPctNoData = (dataStore) =>
```

Comment:
Using mean here because each model and timestamp has its own `pctnd` value for no data.